test/containers/HabitBox.spec.js

```diff
@@ -15,4 +24,30 @@ describe('(Container) HabitBox', () => {
     };
     expect(wrapper.prop('style')).to.eql(expectedStyles);
   });
+
+  it('renders 1 <HabitForm /> component', () => {
+    expect(wrapper.find(HabitForm)).to.have.length(1);
+  });
+
+  it('renders 1 <HabitTabs /> component', () => {
+    expect(wrapper.find(HabitTabs)).to.have.length(1);
+  });
+
+  it('allows us to set state', () => {
+    var schedules =
+      [{
+        "id":5,"habit":
+        {
+        "id":5,"title":"Habit 5","description":"Habit Description 5","created_at":"2016-01-18T20:53:45.932Z","updated_at":"2016-01-18T20:53:45.932Z","goal_id":5,"active":true
+        },
+        "date":"2016-01-18T20:53:45.829Z","status":"do","frequency":"day","repeat":true,"created_at":"2016-01-18T20:53:45Z","updated_at":"2016-01-18T20:53:45Z"
+      }]
+    const wrapper = mount(<HabitBox />);
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>Expected an identifier and instead saw '<'.<br>Expected ')' and instead saw 'HabitBox'.<br>Expected an identifier and instead saw '>'.<br>Too many errors. (83% scanned).